### PR TITLE
#143 improve calendar setting page

### LINKED ISSUE #143: As a user, I want a less confusing behaviour for the Calendar configuration page
#### User story summary

As a user, I want a less confusing behaviour for the Calendar configuration page.

#### Criteria

- [x] The "Cancel" button should be changed to a back arrow icon button (because it only redirects back to the main calendar page and doesn't cancel anything at all).
- [x] There should be a title for the configuration page: "Manage calendars".
- [x] After clicking the "Save" button, I should be redirected back to the main calendar page.
- [ ] If I press the back arrow icon button (currently the "Cancel" button) and there's some changes I haven't saved, there should be a prompt to warn that I have unsaved changes.
- [x] There should be a notification when the configuration has been saved successfully.
- [x] There should be a notification when the configuration fails to be saved.
- [x] The "Save" button should not be active unless I have made some changes.
- [x] The toggles/switches in the "DISPLAY" tab should be replaced with checkboxes (see the screenshots below).
- [ ] (Optional) If I'm already in the Calendar configuration page, when I click "Manage calendars", I will be redirected back to the main calendar page.

#### UI/UX Design

calendar_configuration_desktop_1

calendar_configuration_desktop_2

### PR DESCRIPTION
**Pull request by tuanlc**
Partially resolve #143
- #143 Use back button instead of Cancel button
- #143 add title for manage calendars page
- #143 redirect users to the calendar main page after saving configuration successfully
- #143 require settings-display.less file in settings.less instead of calendar global style file
- #143 Use checkbox inputs instead of toggle/switch inputs
- #143 add missing translations when moving i18n from linagora.esn.calendar to esn.calendar.libs
 - #143 correct french translation 

ezgif com-video-to-gif

### CODE REVIEW

File: src/esn.calendar.libs/i18n/fr.json [hunk 1]
```diff
@@ -129,7 +129,8 @@
   "has proposed changes to the event": "a proposé des changements à l'évènement",
   "has replied \"Maybe\" to this invitation": "a provisoirement accepté cet événement",
   "has updated a meeting": "a mis à jour un événement",
-  "Hide": "Masquer",
+  "Hide non-working days": "Masquer les jours non travaillés",
```
Comment (rezk2ll) [inline src/esn.calendar.libs/i18n/fr.json:132]:
```suggestion
  "Hide non-working days": "Masquer les jours non ouvrés",
```

File: src/linagora.esn.calendar/app/settings/subheader/settings-subheader.pug [hunk 2]
```diff
@@ -3,5 +3,6 @@ extends /modules/subheader/responsive-subheader.pug
 block left
   button.btn.btn-link(ui-sref="calendar.main")
     .flex-vertical-centered.flex-space-between
-      i.mdi.mdi-close
-      .hidden-xs.vertical-centered {{ 'Cancel' | translate }}
+      i.mdi.mdi-arrow-left
+  .title
+    | {{ 'Manage calendars' | translate }}
```
Comment (rezk2ll) [inline src/linagora.esn.calendar/app/settings/subheader/settings-subheader.pug:8]:
```suggestion
    | {{ 'Manage calendars' | translate }}

```

File: src/esn.calendar.libs/i18n/en.json [hunk 3]
```diff
@@ -117,6 +117,7 @@
   "Export your calendar to iCalendar format file (.ics) which you can import to other applications.": "Export your calendar to iCalendar format file (.ics) which you can import to other applications.",
   "Failed to create resource": "Failed to create resource",
   "Failed to retrieve resources details": "Failed to retrieve resources details",
+  "Failed to submit request":"Failed to submit request",
```
Comment (huy-ta) [inline src/esn.calendar.libs/i18n/en.json:120]:
```suggestion
  "Failed to submit request": "Failed to submit request",
```

**Review by rezk2ll — DISMISSED** [review]

**Review by huy-ta — APPROVED** [review]

**Comment by huy-ta** [comment]
Wait, the notification after saving the configuration is not translated?

**Comment by huy-ta** [comment]
And also the "waves" effect around the icon button should be round/circle, not a square.

**Review by rezk2ll — APPROVED** [review]

**Comment by tuanlc** [comment]
> Wait, the notification after saving the configuration is not translated?

Added

> And also the hover effect around the icon button should be round/circle, not a square.

Done

**Review by huy-ta — APPROVED** [review]

**Review by asmaNode — DISMISSED** [review]

**Review by rezk2ll — DISMISSED** [review]

**Review by huy-ta — APPROVED** [review]

**Review by asmaNode — DISMISSED** [review]

**Review by rezk2ll — APPROVED** [review]

File: src/esn.calendar.libs/i18n/vi.json [hunk 4]
```diff
@@ -275,5 +279,8 @@
...
   "Search in events": "Tìm sự kiện",
-  "Forwarding this invitation could allow any recipient to send a response to the organizer and be added to the guest list, or invite others regardless of their own invitation status, or to modify your RSVP.": "Nhờ chuyển tiếp lời mời này, mọi người nhận có thể gửi câu trả lời cho người tổ chức và được thêm vào danh sách khách hoặc mời những người khác bất kể trạng thái lời mời của họ hoặc sửa đổi câu trả lời."
+  "Forwarding this invitation could allow any recipient to send a response to the organizer and be added to the guest list, or invite others regardless of their own invitation status, or to modify your RSVP.": "Nhờ chuyển tiếp lời mời này, mọi người nhận có thể gửi câu trả lời cho người tổ chức và được thêm vào danh sách khách hoặc mời những người khác bất kể trạng thái lời mời của họ hoặc sửa đổi câu trả lời.",
+  "Saving configuration...": "Đang lưu cấu hình...",
+  "Configuration saved": "Đã lưu cấu hình",
+  "Failed to save configuration": "Không thể  lưu cấu hình"
```
Comment (piavgh) [inline src/esn.calendar.libs/i18n/vi.json:285]:
```suggestion
  "Failed to save configuration": "Không thể lưu cấu hình"
```

File: src/esn.calendar.libs/i18n/vi.json [hunk 5]
```diff
@@ -275,5 +279,8 @@
...
   "An error occured on our side, please contact administrator if problem persists": "Đã xảy ra lỗi ở phía chúng tôi, vui lòng liên hệ với quản trị viên nếu sự cố vẫn tiếp diễn",
   "Search in events": "Tìm sự kiện",
-  "Forwarding this invitation could allow any recipient to send a response to the organizer and be added to the guest list, or invite others regardless of their own invitation status, or to modify your RSVP.": "Nhờ chuyển tiếp lời mời này, mọi người nhận có thể gửi câu trả lời cho người tổ chức và được thêm vào danh sách khách hoặc mời những người khác bất kể trạng thái lời mời của họ hoặc sửa đổi câu trả lời."
+  "Forwarding this invitation could allow any recipient to send a response to the organizer and be added to the guest list, or invite others regardless of their own invitation status, or to modify your RSVP.": "Nhờ chuyển tiếp lời mời này, mọi người nhận có thể gửi câu trả lời cho người tổ chức và được thêm vào danh sách khách hoặc mời những người khác bất kể trạng thái lời mời của họ hoặc sửa đổi câu trả lời.",
+  "Saving configuration...": "Đang lưu cấu hình...",
+  "Configuration saved": "Đã lưu cấu hình",
```
Comment (piavgh) [inline src/esn.calendar.libs/i18n/vi.json:284]:
```suggestion
  "Configuration saved": "Cấu hình đã được lưu",
```

Comment (tuanlc) [reply src/esn.calendar.libs/i18n/vi.json:284]: They are the same

**Review by piavgh — APPROVED** [review]

**Review by rezk2ll — APPROVED** [review]

**Pull request merged**